bundles/binding/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/internal/GPIOBinding.java

```diff
@@ -86,8 +86,10 @@ protected void internalReceiveCommand(String itemName, Command command) {
 						if (gpioPin != null) {
 							if (command == OnOffType.ON) {
 								gpioPin.setValue(GPIOPin.VALUE_HIGH);
+								logger.debug("Send command 1 to GPIO " + gpioPin.getPinNumber().toString());

```

Comment:
Doesn't `GPIOPin.getPinNumber()` return an `int`, which is a primitive type and therefore has no methods like `toString()`?  Also, if you were to use parameterized arguments to logger.debug(), you would avoid creating string objects when DEBUG logging is not enabled, like `logger.debug("Send command 1 to GPIO {}",  gpioPin.getPinNumber());` (this would work because the `int` primitive will get autoboxed to `java.lang.Integer`)